fitsio/test.py

```diff
@@ -1296,8 +1295,19 @@ def testGZIPTileCompressedReadLosslessAstropy(self):
         """
         Test reading an image gzip compressed by astropy (fixed by cfitsio 3.49)
         """
-        gzip_file = resource_filename(__name__, 'test_images/test_gzip_compressed_image.fits.fz')
-        data = fitsio.read(gzip_file)
+        try:
+            import importlib.resources
+            ref = importlib_resources.files("fitsio") / 'test_images' / 'test_gzip_compressed_image.fits.fz'
```

Comment:
If we ever want to support windows, we should be using path joining
```
import os
path = os.path.join(
    importlib_resources.files("fitsio"),
    'test_images',
    'test_gzip_compressed_image.fits.fz',
)
```

Reply:
This is pathlib. The / is an overloaded python operator, not a character. This is equivalent to os.path.join

Reply:
Thanks, sorry about that. I was mentally thinking of the code Mike posted `gzip_file = Path(__file__).parent / 'test_images/test_gzip_compressed_image.fits.fz'
`

Reply:
That works the same way I believe.

Reply:
Except it should be `gzip_file = Path(__file__).parent / 'test_images' / 'test_gzip_compressed_image.fits.fz'`